Assets/Mirror/Examples/Lobby/Scripts/OfflineGUI.cs

```diff
@@ -14,7 +14,7 @@ void Start()
             Camera.main.enabled = true;
 
             // Since this is a UI only screen, lower the framerate and thereby lower the CPU load
-            Application.targetFrameRate = 10;
+            QualitySettings.vSyncCount = 4;
```

Comment:
why is this still in here at all?
and why vsync count = 4?
I really don't think we should mess with the user's frame rate. this can cause unwanted side effects, especially in VR.

Reply:
This is just an exemple no ?
Developpers can just modify it for their needs (if they copy/re take the exemple)

Reply:
That's in the example, not the core.  That offline scene has nothing but a button.

Reply:
I really think we should remove this and let the mirror users worry about this.
as long as the lobby example shows how to use the lobby, that's fine :)

Reply:
This really does not belong here.  People that read this example and see this will just wonder if this is necessary.   

As an example,  I think we need to be minimal,  just show only what is strictly required to get it to work

Reply:
Removed